scripts/Packages/Package_Gullwing__QFP_SOIC_SO/Readme.md

```diff
@@ -62,8 +63,8 @@ _Note: Contributions intended for the official library shall not include the man
 
 ### Pad count
 - Pad count (`num_pins_x`, `num_pins_y`) {int}
-  - `num_pins_x`=0 is used for generating SOIC like packages.
-  - `num_pins_y`=0 is used to generate SOIC like package footprints but with inverted pin numbering (Mirrored numbering scheme. Some manufactures use this style in their datasheets. Make sure you are not looking at the bottom view before using this. Not supported for QFP and similar.)
+  - `num_pins_x`=0 is used for generating SOIC-like packages.
+  - `num_pins_y`=0 is used to generate SOIC-like package footprints but with inverted pin numbering (Mirrored numbering scheme. Some manufactures use this style in their datasheets. Make sure you are not looking at the bottom view before using this. Not supported for QFP and similar.)
```

Comment:
Missing a period before the open paren. There are a lot of little sentence in here which might be nice to chain together, especially the first two. Also, an image of the resulting footprint and link to a datasheet might make the result of this option much more clear.)

Reply:
fixed